wBoooksApi/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PostsController < ApplicationController
```

Comment:
Hmm why did you add this controller? We aren't using the Post controller or model at all

Reply:
By the way, we try to not use the scaffold if possible, it creates a lot of things that probably you aren't using

Reply:
I ran an scaffold as indicated in the guide provided in the link of the training. Obviously didnt have to do this step